grazer/src/main/webapp/WEB-INF/views/personGraph.jsp

```diff
@@ -25,6 +25,9 @@
     <c:when test="${edge.targetNode.type == 'http://www.digitalhps.org/types/TYPE_986a7cc9-c0c1-4720-b344-853f08c136ab' }" >
         <a href="<c:url value="/concept/${edge.targetNode.conceptId}" />">${edge.targetNode.label}</a>
     </c:when>
+    <c:when test="${edge.targetNode.type != 'http://www.digitalhps.org/types/TYPE_986a7cc9-c0c1-4720-b344-853f08c136ab' }" >
+        <a href="<c:url value="/concept/${edge.targetNode.conceptId}" />">${edge.targetNode.label}</a>
+    </c:when>
```

Comment:
The condition edge.targetNode.type != 'http://www.digitalhps.org/types/TYPE_986a7cc9-c0c1-4720-b344-853f08c136ab' includes **all the edges whose target node type is not** 'http://www.digitalhps.org/types/TYPE_986a7cc9-c0c1-4720-b344-853f08c136ab'.  Isnt' the c:otherwise tag sufficient to handle this case ?

Reply:
Yes, I made the change